backbone/src/main/res/values/styles.xml

```diff
@@ -138,6 +138,14 @@
         <item name="android:layout_width">100dp</item>
         <item name="android:layout_height">100dp</item>
         <item name="android:layout_marginStart">16dp</item>
+        <item name="android:scaleType">centerCrop</item>
```

Comment:
What does this fix exactly?

Reply:
Normally (and you can tell me if i am wrong) this will avoid to resize the capture. You will only get the centre and crop everything else. Certainly to avoid distorted image ?

Reply:
@jaimesDSG I won't correct you because you're not wrong :) For the record, I know what centerCrop does, and what all the other scaleTypes do. My question is: what is this fixing *here*. This line is added with no mention in the PR description, so I'm asking: why is this needed, what is this fixing, and why CROPPING (therefore LOSING) part of a picture?

Reply:
I'm cropping this because I need to add the tiny square icon in the top right corner, If I don't crop, I won't be able to know the exact view size and the square icon will not be aligned correctly. if you have a better way to do this please let me know

Reply:
So I [asked](https://app.zeplin.io/project/5d8e16c2db3c0d61aac43a5f/screen/5da0bfe52eea29085f322710) Felix from UI on Zeplin on the best way to show the maximize button, and we agreed I should crop the image and fill the view size